README.md

```diff
@@ -25,11 +25,11 @@ $ go get github.com/garyburd/redigo/redis
 ```
 $ ./update_deps.sh $buildtype $extra_cmake_flags
 ```
-   $buildtype is optional; it defaults to Release and may be set to any LLVM
+   `$buildtype` is optional; it defaults to Release and may be set to any LLVM
    build type.
-   $extra_cmake_flags is optional. It is passed to CMake.
+   `$extra_cmake_flags` is optional. It is passed to CMake.
 
-2. Run CMake from a build directory:
+2. Run `CMake` from a build directory:
```

Comment:
CMake is just the name of the tool, and not a command that can be executed, so should not be quoted.